Tests/UnitTests/CustomerCenter/Events/CustomerCenterEventsRequestTests.swift

```diff
@@ -31,15 +31,16 @@ class CustomerCenterEventsRequestTests: TestCase {
         let storedEvent: StoredEvent = try XCTUnwrap(.init(event: event,
                                                            userID: Self.userID,
                                                            feature: .customerCenter,
-                                                           appSessionID: Self.appSessionID))
-        let requestEvent: EventsRequest.CustomerCenterEvent = try XCTUnwrap(.init(storedEvent: storedEvent))
+                                                           appSessionID: Self.appSessionID,
+                                                           eventDiscriminator: "lifecycle"))
```

Comment:
Can potentially use the new discriminator enum?

Reply:
yes. Thanks!